package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "root",
   "private": true,
+  "scripts": {
+    "prepare": "git config core.hooksPath .githooks"
+  },
```

Comment:
Don't know why github is showing these stuff related to linter hook, since they were already merged into master.